src/features/tokens/routes.ts

```diff
@@ -1,35 +1,36 @@
 import { useQuery } from '@tanstack/react-query';
 import { useMemo } from 'react';
 
+import { HypERC20Collateral, TokenType } from '@hyperlane-xyz/hyperlane-token';
 import { utils } from '@hyperlane-xyz/utils';
 
 import { areAddressesEqual, isValidAddress, normalizeAddress } from '../../utils/addresses';
 import { logger } from '../../utils/logger';
 import { getErc20Contract } from '../contracts/erc20';
-import { getHypErc20CollateralContract } from '../contracts/hypErc20';
+import { getHypWrapperContract } from '../contracts/hypErc20';
 import { getProvider } from '../multiProvider';
 
 import { getAllTokens } from './metadata';
 import { TokenMetadata, TokenMetadataWithHypTokens } from './types';
 
 export enum RouteType {
-  NativeToRemote = 'nativeToRemote',
+  BaseToRemote = 'baseToRemote',
```

Comment:
nit: `base` is new terminology to me, usually we talk about local/remote or origin/destination.

Reply:
Yes but those terms are relative. I need a term that refers to the original source of the token. Totally open to renaming if you have a preferred alternative?